src/tokens/component/contribution.json5

```diff
@@ -0,0 +1,340 @@
+{
+  contribution: {
+    default: {
+      bgColor: {
+        '0': {
+          $value: '{base.color.neutral.2}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+            'org.primer.overrides': {
+              'dark-high-contrast': '{base.color.neutral.3}',
+              'light-high-contrast': '{base.color.neutral.3}',
+              'light-dimmed': '{base.color.neutral.5}',
+            },
+          },
+        },
+        '1': {
+          $value: '{base.color.green.1}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+            'org.primer.overrides': {
+              dark: '{base.color.green.8}',
+            },
+          },
+        },
+        '2': {
+          $value: '{base.color.green.3}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+            'org.primer.overrides': {
+              dark: '{base.color.green.6}',
+            },
+          },
+        },
+        '3': {
+          $value: '{base.color.green.4}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+          },
+        },
+        '4': {
+          $value: '{base.color.green.6}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+            'org.primer.overrides': {
+              dark: '{base.color.green.2}',
+            },
+          },
+        },
+      },
+      borderColor: {
+        '0': {
+          $value: '{base.color.black}',
+          alpha: 0.05,
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              group: 'component (internal)',
+              scopes: ['bgColor'],
+            },
+            'org.primer.overrides': {
+              dark: {
+                $value: '{base.color.white}',
+                alpha: 0.05,
+              },
+              'dark-dimmed': {
+                $value: '{base.color.white}',
+                alpha: 0.05,
+              },
+              'dark-high-contrast': {
+                $value: '{base.color.white}',
+                alpha: 1,
+              },
+              'light-high-contrast': {
+                alpha: 1,
+              },
```

Comment:
How does this look @lukasoppermann? Can I just override alpha like I've done for light high contrast?

Also, you may notice all of these border colors are the same (so why do we need individual tokens?) I plan to override them individually to achieve 3:1 contrast behind our border contrast feature flag. I'm not sure what the token structure will look like for that yet but I know I need custom values for each level 😄

Reply:
Lookinh at the output it seems like you need to also add the `$value` token. This is something we could possibly optimize though.